distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response.enso

```diff
@@ -162,3 +173,28 @@ type Response
         type_pair = ["type", "Response"]
         cons_pair = ["constructor", "Value"]
         JS_Object.from_pairs [type_pair, cons_pair, ["headers", self.headers], ["body", self.body], ["code", self.code]]
+
+    ## PRIVATE
+    get_header : Text -> Text | Nothing
+    get_header self name:Text =
```

Comment:
Can we follow the `get` style here and have a `(~default=Nothing)`?
Allows us to put defaults in or error then as needed.